qa-core/src/tests/public-api/tables/tables.spec.ts

```diff
@@ -0,0 +1,48 @@
+import TestConfiguration from "../TestConfiguration"
+import PublicAPIClient from "../PublicAPIClient"
+
+describe("Public API - /tables endpoints", () => {
+  let api: PublicAPIClient
+  const config = new TestConfiguration()
+
+  beforeAll(async () => {
+    await config.beforeAll()
+    const app = await config.seedApp()
+    api = new PublicAPIClient(app.data._id)
+  })
+
+  afterAll(async () => {
+    await config.afterAll()
+  })
+
+  it("POST - Create a table", async () => {
+    const response = await api.post(`/tables`, {
+      body: require("./fixtures/table.json")
+    })
+    const json = await response.json()
+    config.testContext.table = json.data
+    expect(response).toHaveStatusCode(200)
+  })
+
+  it("POST - Search tables", async () => {
+    const response = await api.post(`/tables/search`, {
+      body: {
+        name: config.testContext.table.name
+      }
+    })
+    expect(response).toHaveStatusCode(200)
```

Comment:
Should we be matching on the response content in these tests? Or should we try to start enforcing that at the unit test level instead? It seems like something we are missing, we assert on the status only in the majority of the unit tests inside worker and server

Reply:
I think we should - I was just mapping these to be exactly like newman so we could get jest integrated